src/rgw/CMakeLists.txt

```diff
@@ -177,12 +177,13 @@ endif (WITH_RADOSGW_BEAST_FRONTEND)
 
 add_library(radosgw_a STATIC ${radosgw_srcs}
   $<TARGET_OBJECTS:civetweb_common_objs>)
+target_link_libraries(radosgw_a PRIVATE rgw_a)
 if (WITH_RADOSGW_BEAST_FRONTEND AND WITH_RADOSGW_BEAST_OPENSSL)
-  target_link_libraries(radosgw_a rgw_a ${SSL_LIBRARIES})
+  target_link_libraries(radosgw_a ${SSL_LIBRARIES})
 endif()
 
 add_executable(radosgw rgw_main.cc)
-target_link_libraries(radosgw radosgw_a librados
+target_link_libraries(radosgw radosgw_a rgw_a librados
```

Comment:
can you please drop this change, now that radosgw gets rgw_a via radosgw_a?

Reply:
@cbodley Done